app/build.gradle.kts

```diff
@@ -218,7 +218,7 @@ android {
     defaultConfig {
         applicationId = "openfoodfacts.github.scrachx.openfood"
 
-        minSdk = 21
+        minSdk = 24
```

Comment:
Is this mandatory for upgrading the AGP?

Reply:
It is not! Apologies I had meant to revert this according to our previous discussion, it will be included in next commit to resolve some language translations linting errors.